### PR TITLE
[IMP] inventory: update integrating_landed_costs

### PR DESCRIPTION
**Pull request by brse-odoo**
This PR is a **targeted revision** for **V14.0 only**.

Performed complete rewrite for Inventory document: integrating_landed_costs.rst

-Added new sections to document
-Fixed and replaced all screenshots
-Removed use of 2nd person
-Reworked example flow in doc

Cleaned up RST, including:
-Fixed 100th character line breaks
-Fixed indentations
-Fixed typos and improved grammar

### CODE REVIEW

**Comment by brse-odoo** [comment]
@StraubCreative, this is ready for your review whenever you have a chance. Thank you!

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @brse-odoo 👋 Ive made some technical changes on this doc, stated below for your review. I will be pushing these changes in a second commit for your approval, once approved I will squash the commits, rebase, and push them back up for final review.

This doc is written very well, my only changes were on 2 things:

1. :guilabel: use on general terms like "landed costs", "vender bills", and "inventory valuation"
2. Indenting within admonition blocks.

Please let me know if you approve of these changes and Ill forward this onto final review. Thank you 🙂

**Comment by brse-odoo** [comment]
> Hi @brse-odoo 👋 Ive made some technical changes on this doc, stated below for your review. I will be pushing these changes in a second commit for your approval, once approved I will squash the commits, rebase, and push them back up for final review.
> 
> This doc is written very well, my only changes were on 2 things:
> 
> 1. :guilabel: use on general terms like "landed costs", "vender bills", and "inventory valuation"
> 2. Indenting within admonition blocks.
> 
> Please let me know if you approve of these changes and Ill forward this onto final review. Thank you 🙂

Hi @samueljlieber, thanks for completing the technical review and implementing these changes! Everything looks good to me and you have my approval on the changes.

**Comment by samueljlieber** [comment]
Hi @brse-odoo, thanks for approving 🙂 I squashed the changes and rebased in https://github.com/odoo/documentation/commit/a825b6a33664dc54b4d42b83480458290abcfc65.

All set for your final review @StraubCreative 😎

**Review by samueljlieber — APPROVED** [review]

**Comment by brse-odoo** [comment]
Hi @odoo/us-doc-review, this PR could use a content review whenever one of you has a chance. Thank you!

**Sending through a content review check before sending to zst for final review. Will squash commits/rebase/push up changes and re-request final review when ready.**

File: content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst:92]: I can see these columns (Original Value and New Value), but everything under the Valuation Adjustments tab is blank. I've made sure to use a FIFO product, etc., but perhaps I did something incorrect along the way? It's worth looking into - and I'm happy to give this workflow another try if something needs to be corrected/adjusted.

File: content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst:99]: When I attempted to click 'Validate' I got the following error message: You cannot apply landed costs on the chosen Transfers(s). Landed costs can only be applied for products with FIFO or average costing method.

However, I made sure that FIFO was applied to the Product Category that was applied to BOTH: the product initially added to the RfQ & to the Service Type 'Landed Cost' product added during the Vendor Bill Draft page section. So, perhaps this is a bug? Did I misunderstand a step along the way? Worth looking into either way.

File: content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst:107]: Was unable to reach this portion, as the above steps caused an error pop-up.

**Review by ksc-odoo — COMMENTED** [review]
Hey @brse-odoo just finished a peer review of this doc, but I ran into some issues near the end of the flow. If you could give those steps a quick "double-check" that would be great. If I made an error along the way, or if a change was made, just let me know and I'll happily give this another try. Just let me know ... Other than that - great work!

File: content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst:58]: 100th character

File: content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst:62]: 100th character

File: content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst:75]: 100th character

File: content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst:95]: 100th character

File: content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst:109]: 100th character

File: content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst:123]: 100th character

File: content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst:131]: 100th character

File: content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst:134]: 100th character

File: content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst:137]: 100th character

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hi @brse-odoo 
Please fix all of the 100th character line break issues and tag me again for another review.
These should have not made it past technical review 😉 
Thank you.

cc: @samueljlieber

**Comment by brse-odoo** [comment]
Removed file `product_variants` in latest commit `4731e2e` to resolve merge conflicts/ci check failures. Pushed up changes and will review doc once more before sending to review.

**Comment by brse-odoo** [comment]
Hi @StraubCreative this PR is ready for final review when you have the chance, thank you 👍

**Review by StraubCreative — APPROVED** [review]
Hi @brse-odoo 
Good PR. 
Approving with comments for you to address.
When you're done, you can run the merge command.
Thanks!

@fw-bot ignore
@robodoo delegate=brse-odoo

**Pull request closed without merging**
